observability/cockpit/concepts.mdx

```diff
@@ -24,6 +24,10 @@ Alerting detects complex conditions defined by a rule and keeps you aware of iss
 
 Scaleway's alert manager allows you to manage and respond to alerts. It handles alerts sent when the alerting rules we run are firing. The alert manager triggers alerts (e.g. emails or texts) if a criteria you have configured on your applications' metrics and logs is activated.
 
+<Message type="tip">
+Select **Scaleway Alerting** when choosing the Alertmanager if you want to manage your alerts using Grafana.
```

Comment:
Is it right that there's no space in Alertmanager?

Reply:
That's how it displays in Grafana, I don't mind putting a space though ![Capture d’écran 2023-01-25 à 16 28 03](https://user-images.githubusercontent.com/87707325/216055557-5e9c5af0-377c-4811-a172-e31898217067.png)